frontend/app/controllers/TierController.scala

```diff
@@ -156,7 +156,7 @@ object TierController extends Controller with ActivityTracking
 
     // Preselect the country from Identity fields
     // but the currency from Zuora account
-    def pageInfo(pf: PrivateFields, billingPeriod: BillingPeriod): PageInfo = {
+    def getPageInfo(pf: PrivateFields, billingPeriod: BillingPeriod): PageInfo = {
       val selectedCountry = pf.billingCountry.orElse(pf.country).flatMap { name =>
         CountryGroup.countries.find(_.name == name)
```

Comment:
If this is being used by multiple controllers I'd put it in some function somewhere common rather than having controllers depend on other controllers, maybe in the views.support.PageInfo?